ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+_Imprinting and Merging surfaces accelerates the DAGMC process by combining surfaces._
+
+  1. Imprint and Merge all shared surfaces in the model.
+     * In command panel select __Geometry -> Volume -> Imprint Merge__ and select __Imprint and Merge__ from the dropdown menu. Then type "all" into the __Volume ID(s)__ box and click __Apply__.
+     * In command line type `imprint volume all` and then enter. Then type `merge volume all`.
+     * _Now all surfaces that previously overlapped have been merged into one surface shared by both volumes._
+
+<details><p><summary>Notes</summary>
+- The default tolerated distance for imprinting/merging surfaces is 5.0e-4 units.
+<br/>
+- Combining surfaces cuts two surfaces and two sets of tetrahedron mesh areas down to one, decreasing computational power required as well as eliminating possible errors where particles exist simultaneously in two surfaces.
+</p></details>
+
+#### Assigning Materials in Cubit/Trelis
+_Most commands for this section are best implemented through the command line. More general instructions can
+be found_ [here](http://svalinn.github.io/DAGMC/usersguide/codes/dag-mcnp.html).
+
+  1. The general command for assigning Materials/Densities in Cubit/Trelis
+  `group "mat:[assigned material number]/rho:[assigned density]" add
+  vol [volume numbers]` where the density is in [atoms/barn-cm] for positive values and [g/cc] for negative values.
+  2. The blocks will be given two different materials and then the void and the graveyard volumes will be assigned. These material assignments will depend on the material library you create/use for the project.
+     * To assign one block to the first material use `group "mat:1/rho:0.0223" add vol 1`
+     * To assign the rest of the blocks to the second material use `group "mat:2/rho:0.03" add vol 2 to 4`
+     * To assign the void volume to vacuum use `group "mat:Vacuum" add vol 5`
+     * Finally to assign the graveyard volume use `group "mat:Graveyard" add vol 6`
+
+#### Exporting file to .h5m format
+_This plugin translates the CAD file to something that can be used by DAGMC. Instructions as to how to install this plugin can be found_ [here](https://svalinn.github.io/DAGMC/install/plugin.html).
+
+  1. Once the plugin is correctly installed you should receive the message `Loaded Svalinn plugin`, line break, `DAGMC export command available` in the command line.
+  2. Once all issues are resolved in the model, export the file as .h5m, this command must be executed through the command line.
+     * The general form of the export command is `export dagmc "<filename>" [options]`. To see the options available enter `help export dagmc` into the command line.
+     * The filename should include the filepath after the working directory so that it designates where the file ought to be saved after export.
+     * For this part `export dagmc "Desktop/ErrorBlocks.h5m"` will create the part as a .h5m with default settings in the Desktop.

```

Comment:
To be safe, this should specify absolute path. I think it assumes relative path wrt where Trelis was launched from, so it can be easy to lose track of the file after exporting.